otx/algorithms/segmentation/tasks/train.py

```diff
@@ -152,6 +152,11 @@ def _init_train_data_cfg(self, dataset: DatasetEntity):
                 ),
             )
         )
+        if len(dataset.get_subset(Subset.UNLABELED)):
```

Comment:
Thanks for applying! Actually, my intention is making as function not only dataset.get_subset code but also the whole of if statement.
as you know, it is duplicated in all three tasks. so, if whole part is made as function, then remove more duplication.

Reply:
Hi, but it is my view that making this whole if-statement as a function is not good. Because this is not a final implementation for semi-dataset so we don't know what args are needed for unlabeled mmdataset for each tasks at the last. So I prefer to leave it as is to make it modify easily by this train_cfg in future implementation. And just making a function just for unlabeled cfg is not appropriate and looks better when we declare them inside init_train_data_cfg.

Reply:
I got it. Maybe moving "_init_train_data_cfg" to parent class can be better choice. but it's out of this score, I think.